src/demo/components/launcher/AppCard.tsx

```diff
@@ -6,11 +6,12 @@ import {Application} from '../../../client/directory';
 
 interface AppCardProps {
     app: Application;
+    isDirectoryApp?: boolean;
```

Comment:
Suggest drop the `?`, as it's not obvious what `undefined` would be in this case